src/index.js

```diff
@@ -7,6 +7,7 @@
  * [![Build Status](https://travis-ci.org/nxus/pipeliner.svg?branch=master)](https://travis-ci.org/nxus/pipeliner)
  * 
  * A framework for creating and running data pipelines.  Data pipelines have stages, which are made of an arbitrary number of tasks.  Stages and tasks are run in serial: once a task completes, the next task in the pipeline is executed.  
+ * A task may complete synchronously, or asynchronously through use of a promise. 
```

Comment:
The docs don't provide enough information to use a pipeline without reading the code, if you're starting from "I have no idea what this does." To get oriented, it's helpful to know it serializes asynchronous tasks as well as synchronous, and that it's based on promises.